### DISCUSSION TITLE
Getting an error trying to run curl

### DISCUSSION THREAD

**Discussion by npentell** [hyperion-project/hyperion.ng#1331] [Q&A]
I've been searching everywhere for info on how to use the json features and found very little info.  I've also made several posts in the forum with no responses so I thought I might try here. 

I am attempting to set something up where my LED lights turn on and off with the TV.  I have set up a curl script but am getting an error.  I'm running Windows 10.  The curl is this

curl -X POST -i "http://192.168.1.247:8090/json-rpc" -d '{"command":"componentstate","componentstate":{"component":"ALL","state":false}}'

I'm getting an error that says 

[hyperiond HTTPJSONRPC] (ERROR) Failed to parse json data from JsonRpc@::ffff:192.168.1.64: Error: illegal value at Line: 0, Column: 1

it's pretty vague to me since everything in on line 0.  Especially, since i'm not very familiar with json and curl to begin with.

any help would be appreciated.

**Comment by Lord-Grey** [comment]
@npentell Your command is correct. It seems there is a problem with the rpc service.
I tried myself too and in rare circumstances, the command does not work as expected.
I suggest you restart hyperion and re-run the command again.

**Reply by npentell** [reply]
That makes sense because it seemed to run the first time I tried it but never ran again after that.  I will try restarting.  Do you know of any solutions in the works?
